diff --git a/shared/index.tsx b/shared/index.tsx
index c803075e517ef5f10cc39a00daab29e8c553a61e..dbef8e40f0b26feeb188f331df0787296df5ef40 100644
--- a/shared/index.tsx
+++ b/shared/index.tsx
@@ -2,4 +2,4 @@ export { default as clientOnly } from "./clientOnly";
 export { FileRoutes } from "./FileRoutes";
 export { HttpHeader } from "./HttpHeader";
 export { HttpStatusCode } from "./HttpStatusCode";
-
+export { apiRoutes } from "./routes";
diff --git a/shared/routes.ts b/shared/routes.ts
index 4b8def9174fb6d77402548a5389987f861c5cc8c..7628ba6678ff530e2afebd150c78992fded2db7d 100644
--- a/shared/routes.ts
+++ b/shared/routes.ts
@@ -30,7 +30,7 @@ declare module "vinxi/routes" {
 }
 
 export const pageRoutes = defineRoutes((fileRoutes as unknown as Route[]).filter(o => o.type === "page"));
-const apiRoutes = defineAPIRoutes((fileRoutes as unknown as Route[]).filter(o => o.type === "api"));
+export const apiRoutes = defineAPIRoutes((fileRoutes as unknown as Route[]).filter(o => o.type === "api"));
 
 export function matchAPIRoute(path: string, method: HTTPMethod) {
   const segments = path.replace(import.meta.env.SERVER_BASE_URL, "").split("/").filter(Boolean);
